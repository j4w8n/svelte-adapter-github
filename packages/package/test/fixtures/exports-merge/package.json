{
	"name": "exports-replace",
	"private": true,
	"version": "1.0.0",
	"description": "merges exports field",
	"exports": {
		".": {
			"import": "./index.js"
		},
		"./constants": "./internal/constants.js",
		"./Test": "./Test.svelte"
	}
}
