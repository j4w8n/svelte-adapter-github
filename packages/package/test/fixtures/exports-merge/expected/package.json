{
	"name": "exports-replace",
	"private": true,
	"version": "1.0.0",
	"description": "merges exports field",
	"exports": {
		"./internal/constants": "./internal/constants.js",
		"./internal/Test.svelte": "./internal/Test.svelte",
		"./hello-world": "./hello-world.js",
		"./icons123": "./icons123.js",
		"./Test.svelte": "./Test.svelte",
		"./why_underscore": "./why_underscore.js",
		".": {
			"import": "./index.js"
		},
		"./constants": "./internal/constants.js",
		"./Test": "./Test.svelte",
		"./package.json": "./package.json"
	},
	"type": "module",
	"svelte": "./index.js"
}